### PR TITLE
MA-4171 - Created a new input file for UKGC, with regions updated.

### PR DESCRIPTION
**Pull request by nathantym**
Created a new input file for UKGC, with regions updated.

There is still work to be done in regards to onboarding, especially in regards to how the proper input file is chosen when the onboarding state machine runs. We also need to do regression testing.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,7 +43,9 @@ baseHandler.post = function(params, callback) {
 
   var inputDoc = JSON.parse(fs.readFileSync(__dirname + '/json/state_machine_input.json', {encoding:'utf8'}));
   var ruleJson = JSON.parse(fs.readFileSync(__dirname + '/json/default_config_rules.json', {encoding:'utf8'}));
-  
+
+  params.platform = "ukgc"
```
Comment (mayankbharadwaj) [inline src/index.js]: Will this platform value be the hardcoded value or will be passed to the proxy lambda function from the Sails API framework ?

File: src/json/state_machine_input.json [hunk 2]
```diff
@@ -122,24 +122,24 @@
...
-    "managedVpcPreConfig":{  
+    "managedVpcPreConfig":{
        "account":null,
        "region":null,
-       "regions":[  
-          "us-east-1",
-          "us-west-2"
-       ]
+       "regions": {
+         "g-cloud": ["eu-west-2"],
```
[the remark this answers is not in the document]
Comment (nathantym) [reply src/json/state_machine_input.json]: I refactored this in my newest commit. My intention was to use the factory pattern to set the regions properly. By doing this we can avoid having to overwrite the regions, as well as avoid using large if else chains. I believe this is the cleanest way of implementing this logic.

File: src/index.js (hunk 1 above)
Comment (nathantym) [reply src/index.js]: This was added for testing and has been removed.

File: src/index.js [hunk 3]
```diff
@@ -55,8 +55,11 @@ baseHandler.post = function(params, callback) {
      "SGID": params.sgid,
      "guid": params.companyguid,
      "checkStatus": true,
+     "platform": params.product
```
Comment (RajkumarShivage) [inline src/index.js]: Change the "platform" to "product"

**Pull request merged**